src/Simulation.cpp

```diff
@@ -84,6 +93,38 @@ void Simulation::run(int steps) {
         roll = atan2(2.0 * (s * z + x * y), -1.0 + 2.0 * (s * s + x * x)) *
                RAD2DEG;
 
+        // simulation completion criteria
+        if ((r_dot.z() < -3.0) && !drogue_deploy) {
+            std::cout << "Drogue: " << std::to_string(tStamp_) << " s"
```

Comment:
This info should be logged with the sim_log->info function, see the logging near the bottom of the function for how it works